packages/input-manager/src/integrations/streamdeck/index.ts

```diff
@@ -91,10 +111,13 @@ export class StreamDeckDevice extends Device {
 				if (!prev) prev = { deltaValue: 0 }
 				return {
 					deltaValue: prev.deltaValue - deltaValue,
+					direction: -1,
 				}
 			})
 
-			this.updateFeedback(id, false).catch((err) => this.logger.error(`Stream Deck: Error updating feedback: ${err}`))
+			this.updateFeedback(id, this.#isButtonDown[id] ?? false).catch((err) =>
+				this.logger.error(`Stream Deck: Error updating feedback: ${err}`)
```

Comment:
There are lots of these, perhaps refactor into a single error-handling function? or possibly change `updateFeedback` to not be an async function that handles it's own errors?